Tests/SPIManifestTests/ManifestTests.swift

```diff
@@ -25,12 +25,14 @@ class ManifestTests: XCTestCase {
     }
 
     func test_encode_manifest() throws {
-        let m = Manifest(builder: .init(configs: [
+        let m = Manifest(metadata: .init(authors: "Christian Noon, Jon Shier and other contributors"), builder: .init(configs: [
```

Comment:
Let's make author test data anonymous throughout all tests. "Author One and Author Two" should be good unless testing the max length of the string. Then some repetitions of "0123456789". 👍